scalafix-core/shared/src/main/scala/scalafix/internal/config/ExplicitResultTypesConfig.scala

```diff
@@ -1,36 +1,37 @@
 package scalafix.internal.config
 
 import metaconfig._
-import MetaconfigPendingUpstream.XtensionConfScalafix
+import metaconfig.generic.Surface
+import metaconfig.annotation._
 
 case class ExplicitResultTypesConfig(
```

Comment:
What about the other rules` configuration?

Reply:
I wanted to keep this PR small, prefer to update the other rule configurations in separate PRs.

Reply:
I think it makes more sense if it's all in one PR. It's confusing to have haft the new way and haft the old way.

Reply:
Fair enough, I'll migrate more configs. I will leave DisableSyntax alone to avoid conflicts with #548